authorizer-app-backend/src/main/java/org/radarbase/authorizer/service/OAuth1RestSourceAuthorizationService.kt

```diff
@@ -0,0 +1,183 @@
+/*
+ *  Copyright 2020 The Hyve
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.radarbase.authorizer.service
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody
+import org.radarbase.authorizer.RestSourceClient
+import org.radarbase.authorizer.RestSourceClients
+import org.radarbase.authorizer.api.RequestTokenPayload
+import org.radarbase.authorizer.api.RestOauth1AccessToken
+import org.radarbase.authorizer.api.RestOauth2AccessToken
+import org.radarbase.authorizer.api.RestSourceUserMapper
+import org.radarbase.authorizer.doa.RestSourceUserRepository
+import org.radarbase.authorizer.doa.entity.RestSourceUser
+import org.radarbase.authorizer.util.OauthSignature
+import org.radarbase.authorizer.util.Url
+import org.radarbase.jersey.exception.HttpBadGatewayException
+import org.radarbase.jersey.exception.HttpBadRequestException
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.time.Instant
+import javax.ws.rs.core.Context
+
+abstract class OAuth1RestSourceAuthorizationService(
+    @Context private val restSourceClients: RestSourceClients,
+    @Context private val httpClient: OkHttpClient,
+    @Context private val objectMapper: ObjectMapper,
+    @Context private val userRepository: RestSourceUserRepository,
+    @Context private val userMapper: RestSourceUserMapper
+    ): RestSourceAuthorizationService {
```

Comment:
Please use the following indentation for class constructors:
```suggestion
    @Context private val userMapper: RestSourceUserMapper,
): RestSourceAuthorizationService {
```